seacrowd/sea_datasets/lio_and_central_flores/processing.py

```diff
@@ -0,0 +1,55 @@
+import re
+
+def parse_text(data):
+    cleaned_data = [re.sub(r'^\ufeff?\d+\t|\b\d+\)\s*', '', re.sub(r'\t+', '\t', text)).replace('\t\n', '').replace('\ufeff', '') for text in data]
+
+    result, sublist = [], []
+    for item in cleaned_data:
+        item = item.replace('\t', ' ')
+        if item != '':
+            sublist.append(item)
+        else:
+            if sublist:
+                result.append(sublist)
+                sublist = []
+
+    if sublist:
+        result.append(sublist)
+
+    lio_data = []
+    eng_data = []
+
+    for i in result:
+      if len(i) == 2:
+        lio_data.append(i[0])
+        eng_data.append(i[1])
+      elif len(i) == 3:
+        lio_data.append(i[0])
+        eng_data.append(i[2])
+      elif len(i) == 4:
+        lio_data.append(i[1])
+        eng_data.append(i[3])
```

Comment:
Missing `else` statement? Can you also explain what's happening here? 🙏 Not sure why we're matching with these arbitrary numbers (i.e,. 2,3,4).